master/reference/public-cloud/aws.md

```diff
@@ -49,74 +49,28 @@ Keep reading for information on when IPIP is required in AWS.
 You can check if your hosts have successfully established BGP sessions with
 one another using `calicoctl node status`.
 
-#### Allow Traffic Within a Single VPC Subnet
+#### Allow Container Traffic
 
-By default, each EC2 instance performs source/destination checks on inbound
-and outbound traffic. This ensures that the instance is the source or
-destination of any traffic it sends or receives.
-
-Since Calico allocates a unique IP address to each container that isn't the IP
-of its host, and since Calico doesn't encapsulate container traffic by default,
-traffic to/from containers will be dropped by the AWS src/dst checks
+By default, AWS drops container traffic with an unrecognized IP.
 
 This reverse-path filtering is enabled by default, but can be disabled on each
-host, allowing Calico traffic to pass through unaffected **between hosts within
-the same VPC subnet.**
-Ensure you follow
+host, allowing Calico traffic to pass unaffected
+**between hosts within the same VPC subnet.** See
 [AWS documentation on disabling source/destination checks](http://docs.aws.amazon.com/AmazonVPC/latest/UserGuide/VPC_NAT_Instance.html#EIP_Disable_SrcDestCheck).
 
 #### Allow Traffic Across Different VPC Subnets
```

Comment:
Similar to the comment above, I think we just need to reorganize it so the user-centric stuff is at the top.

We also want to explain that you can use `ipip` for crossing subnets, but still don't need it for within a single subnet.  Expressing that is a little bit trickier, but @robbrockbank can probably help :)

Maybe something like this:

```
#### Across Different VPC Subnets / VPCs

To use Calico across multiple different VPC subnets or peered VPCs, in addition to disabling 
src/dst checks as described above you must enable ipip encapsulation and outgoing NAT 
on your Calico IP pools.  See the [IP pool configuration reference]() for information on how to 
enable these settings.

By default, Calico's ipip encapsulation applies to all container-to-container traffic.  However,
it is only required on traffic between containers which crosses a VPC subnet boundary.  To 
configure Calico to perform ipip encapsulation only across VPC subnet boundaries:

     ...

Enabling ipip encapsulation allows container-to-container traffic to pass VPC subnet boundaries, while outgoing NAT allows containers to access EC2 instances across VPC subnet boundaries.
```

Soon after v2.0 we plan on having a "switch" on the Calico pool which will do the cross-subnet behavior automatically.  At that point we can tweak this section to recommend that first or exclusively.